recipes/iris-ued/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "5.0.2" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "b35e7c07f36fb36335db67832b6e36b32e048abb744a6a33e8b11cda15a19dfa" %}
+
+package:
+  name: 'iris-ued'
+  version: '{{ version }}'
+
+source:
+  fn: 'iris-ued-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/i/iris-ued/iris-ued-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  skip: True  # [not win]
```

Comment:
you probably want to combine these selectors.  This area is technically a dictionary so you could be clobbering your own skip